app/src/main/java/com/starstruckstech/pgpodandroid/NetworkHelper.kt

```diff
@@ -0,0 +1,7 @@
+package com.starstruckstech.pgpodandroid
+
+class NetworkHelper(private val apiService: ApiServicee) {
+    suspend fun getPosts(): List<Post> {
```

Comment:
use retrofit client and do the API Call and return the corresponding data class object for  a sample API